embedded-compositor/qml/Keyboard.qml

```diff
@@ -33,14 +33,38 @@ Item {
     InputPanel {
         id: inputPanel
         visible: y < parent.height
-        y: active ? parent.height - height : parent.height
         anchors.left: parent.left
         anchors.right: parent.right
-        Behavior on y {
-            NumberAnimation { }
-        }
         onYChanged: Qt.callLater(keyboardContainer.updateTranslate)
         onHeightChanged: Qt.callLater(keyboardContainer.updateTranslate)
         onVisibleChanged: keyboardContainer.updateTranslate()
+        state: "HIDDEN"
```

Comment:
This isn’t necessary because we have a `when` statement below. when `inputPanel.active` we get `VISIBLE`, when `!inputPanel.active` we get `HIDDEN`

Reply:
without this, during the booting the keyboard ist visible for 2 seconds

Reply:
its slides in and than immediately slides out